bitshuffle/h5.pyx

```diff
@@ -65,10 +65,24 @@ H5_COMPRESS_LZ4 = BSHUF_H5_COMPRESS_LZ4
 # Init HDF5 dynamic loading with HDF5 library used by h5py
 if not sys.platform.startswith('win'):
     if sys.version_info[0] >= 3:
-        libname = bytes(h5d.__file__, encoding='utf-8')
+        libs = [bytes(h5d.__file__, encoding='utf-8'),
+                bytes(h5s.__file__, encoding='utf-8'),
+                bytes(h5t.__file__, encoding='utf-8'),
+                bytes(h5p.__file__, encoding='utf-8'),
+                bytes(h5z.__file__, encoding='utf-8')]
```

Comment:
I'd also include `h5py.defs` and `h5py.h5fd`, which are the other two I found that were linked to libhdf5